### ISSUE TITLE
Remove some useless buttons from **Edit instance** -> **Version** menu

### ISSUE THREAD

**Issue by slava110** [PolyMC/PolyMC#1036]
### Role

I play modded MC c:

### Suggestion

Remove `Customize` and `Install mods` buttons from **Edit instance** -> **Version** menu right sidebar. They are redundant. `Customize` action could be triggered when you click `Edit` button for the first time, `Install mods` just redirects you to the same **Mods** menu that is available on the left panel

### Benefit

Easier to navigate if there's less useless buttons in the interface

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

**Comment by Desoroxxx** [comment]
I agree with this, it would make the UX better.

In the version tab I noted that some buttons could be entirely hidden under certain circumstances too:

- `Edit` & `Revert` buttons should be hidden when they are disabled (_when customized has not been pressed_)
- `Install Fabric` & `Install Quilt` buttons should be hidden when disabled (_when your version does not support Fabric and Quilt like 1.12_)

**Comment by TayouVR** [comment]
> * `Install Fabric` & `Install Quilt` buttons should be hidden when disabled (_when your version does not support Fabric and Quilt like 1.12_)

All the modloader install buttons should also be disabled when the mod loader is already present. (e.g. when quilt is installed the `Install Quilt` button would be disabled)

**Comment by Desoroxxx** [comment]
> > * `Install Fabric` & `Install Quilt` buttons should be hidden when disabled (_when your version does not support Fabric and Quilt like 1.12_)
> 
> All the modloader install buttons should also be disabled when the mod loader is already present. (either all of them will be disabled, or the one, that is installed. Not sure if you can e.g. install LiteLoader on top of forge or smth..)

No, since you can actually install multiple mod loader at once by example LiteLoader and Forge

**Comment by TayouVR** [comment]
> > > * `Install Fabric` & `Install Quilt` buttons should be hidden when disabled (_when your version does not support Fabric and Quilt like 1.12_)
> > 
> > 
> > All the modloader install buttons should also be disabled when the mod loader is already present. (either all of them will be disabled, or the one, that is installed. Not sure if you can e.g. install LiteLoader on top of forge or smth..)
> 
> No, since you can actually install multiple mod loader at once by example LiteLoader and Forge

well you can't install any mod loader twice though

let me edit my comment then real quick to include only what would make sense then.

**Comment by DioEgizio** [comment]
> > > > * `Install Fabric` & `Install Quilt` buttons should be hidden when disabled (_when your version does not support Fabric and Quilt like 1.12_)
> > > 
> > > 
> > > All the modloader install buttons should also be disabled when the mod loader is already present. (either all of them will be disabled, or the one, that is installed. Not sure if you can e.g. install LiteLoader on top of forge or smth..)
> > 
> > 
> > No, since you can actually install multiple mod loader at once by example LiteLoader and Forge
> 
> well you can't install any mod loader twice though
> 
> let me edit my comment then real quick to include only what would make sense then.

Tbh, those buttons in those cases can be used as an alternative to "change version", so I think it'd stay like that

**Comment by TayouVR** [comment]
> Tbh, those buttons in those cases can be used as an alternative to "change version", so I think it'd stay like that

I didn't realize they acted that way to begin with, fair then. That makes sense

**Comment by Desoroxxx** [comment]
> Tbh, those buttons in those cases can be used as an alternative to "change version", so I think it'd stay like that

Yeah but then why have two button with the same function one of which is way clearer about what it does to the user.
I can't think of any use case where a user would use `Install Forge` over `Change version` to well change version

**Comment by DioEgizio** [comment]
> Install Fabric & Install Quilt buttons should be hidden when disabled (when your version does not support Fabric and Quilt like 1.12)

they already are afaik
<img width="1188" alt="image" src="https://user-images.githubusercontent.com/83089242/184368008-757f1dd5-d316-49cc-83ff-31d70cfec3fc.png">

**Comment by Desoroxxx** [comment]
> > Install Fabric & Install Quilt buttons should be hidden when disabled (when your version does not support Fabric and Quilt like 1.12)
> 
> they already are afaik

They are not hidden, only grey out this add clutter

**Comment by ZekeZDev** [comment]
#1103 should address the first removal of install mods I will address removing the customise button next.
